community/index.md

```diff
@@ -168,24 +168,39 @@ chime_user:
   alt: "Chime Logo" 
 ---
 
+The Spinnaker OSS project was started at Netflix. Now, it thrives on the contributions of the broader DevOps community who have adopted it as their Continuous Delivery tool. 
+
 ## Engage the community and get support
 
-The Spinnaker OSS project was started at Netflix, and today thrives on the contributions and broad adoption of the public devops community. Here are some ways to engage the community and find help&mdash;join us!
+Here are some ways to engage the Spinnaker community and find help&mdash;join us!
+
+### [Slack](http://join.spinnaker.io)
 
-* [Spinnaker community forum](https://community.spinnaker.io)
+Start here! Many Spinnaker contributors and users are active on the Spinnaker Slack. It's a great place to get answers to questions or start a discussion about a feature or topic. Good places to begin on Slack include the following channels: 
+
+* A SIG channel if there is a relevant one, such as `#sig-security`. All SIG slack channels start with the prefix `sig`.
```

Comment:
We should include a brief description of what a SIG is before this

Reply:
Added